hotsos/core/host_helpers/cli.py

```diff
@@ -774,35 +791,66 @@ def command_catalog(self):
+                 BinCmd('openstack-hypervisor.'
+                        'ovn-sbctl --no-leader-only show'),
+                 FileCmd('sos_commands/ovn_central/'
+                         'ovn-sbctl_--no-leader-only_show'),
                  FileCmd('sos_commands/ovn_central/'
+                         'openstack-hypervisor.'
                          'ovn-sbctl_--no-leader-only_show'),
                  # sosreport < 4.5
                  FileCmd('sos_commands/ovn_central/ovn-sbctl_show')],
             'ovs_vsctl_get':
                 [BinCmd('ovs-vsctl get {table} {record} {column}',
                         singleline=True),
+                 BinCmd('openstack-hypervisor.'
+                        'ovs-vsctl get {table} {record} {column}',
+                        singleline=True),
                  FileCmd('sos_commands/openvswitch/ovs-vsctl_-t_5_get_'
+                         '{table}_{record}_{column}', singleline=True),
+                 FileCmd('sos_commands/openvswitch/openstack-hypervisor.'
+                         'ovs-vsctl_-t_5_get_'
                          '{table}_{record}_{column}', singleline=True)],
             'ovs_vsctl_list':
                 [BinCmd('ovs-vsctl list {table}'),
+                 BinCmd('openstack-hypervisor.ovs-vsctl list {table}'),
+                 FileCmd('sos_commands/openvswitch/'
+                         'ovs-vsctl_-t_5_list_{table}'),
                  FileCmd('sos_commands/openvswitch/'
-                         'ovs-vsctl_-t_5_list_{table}')],
+                         'openstack-hypervisor.ovs-vsctl_-t_5_list_{table}')],
             'ovs_vsctl_list_br':
                 [BinCmd('ovs-vsctl list-br'),
-                 FileCmd('sos_commands/openvswitch/ovs-vsctl_-t_5_list-br')],
+                 BinCmd('openstack-hypervisor.ovs-vsctl list-br'),
+                 FileCmd('sos_commands/openvswitch/ovs-vsctl_-t_5_list-br'),
+                 FileCmd('sos_commands/openvswitch/openstack-hypervisor.'
+                         'ovs-vsctl_-t_5_list-br')],
             'ovs_appctl':
                 [OVSAppCtlBinCmd('ovs-appctl {command} {flags} {args}'),
+                 OVSAppCtlBinCmd('openstack-hypervisor.'
+                                 'ovs-appctl {command} {flags} {args}'),
                  OVSAppCtlFileCmd('sos_commands/openvswitch/ovs-appctl_'
+                                  '{command}{flags}{args}'),
+                 OVSAppCtlFileCmd('sos_commands/openvswitch/'
+                                  'openstack-hypervisor.ovs-appctl_'
                                   '{command}{flags}{args}')],
             'ovs_ofctl':
                 [OVSOFCtlBinCmd('{command} {args}'),
+                 SunbeamOVSOFCtlBinCmd('{command} {args}'),
+                 OVSOFCtlFileCmd('sos_commands/openvswitch/'
+                                 'ovs-ofctl{ofversion}_{command}_{args}'),
                  OVSOFCtlFileCmd('sos_commands/openvswitch/'
```

Comment:
I guess you could have also created a SunbeamOVSOFCtlFileCmd but after looking again don't think it's necessary.